fastlane/lib/fastlane/action.rb

```diff
@@ -187,4 +187,20 @@ def dedent!
 
     self.gsub!(/^#{first_line_indent}/, "")
   end
+
+  def shellescape
+    if FastlaneCore::Helper.windows?
+      # double quotes have to be doubled
+      self.gsub('"', '""')
```

Comment:
I think this does nothing, as it returns a copy of the string after the substitutions. You have to use `gsub!` to replace it in-place.